tests/TestHarness/Cluster.py

```diff
@@ -241,10 +242,10 @@ def launch(self, pnodes=1, unstartedNodes=0, totalNodes=1, prodCount=1, topo="me
                 return False
             tries = tries - 1
             time.sleep(2)
-
-        cmd="%s -p %s -n %s -d %s -i %s -f %s --unstarted-nodes %s --logging-level %s" % (
+        loggingLevelDictString = json.dumps(self.loggingLevelDict).replace(" ", "")
```

Comment:
You can include the `separators=(',', ':') ` argument to `json.dumps()` and eliminate the need to call `replace()`.